inputfiles/patches/forced-declare.kdl

```diff
@@ -0,0 +1,7 @@
+// Keeping EventListener and EventListenerObject isn't the most elegant way to handle the event listeners, but we need to keep the EventListener as an extendable interface for libraries like angular.
+interface EventListener noInterfaceObject=#true {
+    method callable {
```

Comment:
We already have `interface.anonymouseMethods` so use it with something like `anonymous-method {}`?